minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -26,8 +26,6 @@
 import tachyon.Constants;
 import tachyon.client.TachyonFS;
 import tachyon.conf.TachyonConf;
-import tachyon.exception.AlreadyExistsException;
-import tachyon.exception.OutOfSpaceException;
```

Comment:
In the future, please don't make changes unrelated to the PR, it makes the code review more difficult. (Applies to other areas of the code as well)